modules/cert-manager-istio-csr-uninstalling.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-integrating-istio.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-istio-csr-uninstalling_{context}"]
+= Uninstalling the {cert-manager-operator} with istio-csr 
+
+
+To uninstall the {cert-manager-operator} with Istio-CSR, use the following steps.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have enabled the `Istiocsr`.
+* You have created the `Istiocsr` custom resource.
+
+.Procedure
+
+. Remove the `IstioCSR` custom resource by running the following command:
++
+[source,terminal]
+----
+$ oc -n <istio_csr_project_name> delete istiocsrs.operator.openshift.io default
+----
+
+. Remove related resources by running the following commands
++
+[IMPORTANT]
+====
+To avoid disrupting any {SMProductName} or Istio components, ensure that no component is referencing the Istio-CSR service or the certificates issued for Istio before removing the following resources.
+====
+
+.. Delete the cluster scoped resources listed by running the following command:
++
+[source,terminal]
+----
+$ oc get clusterrolebindings.rbac.authorization.k8s.io,clusterroles.rbac.authorization.k8s.io -l "app=cert-manager-istio-csr,app.kubernetes.io/name=cert-manager-istio-csr"
+----
+
+.. Delete the resources in Istio-csr deployed namespace, listed by running the following command:
++
+[source,terminal]
+----
+$ oc get certificate.cert-manager.io,deployments.apps,services,serviceaccounts -l "app=cert-manager-istio-csr,app.kubernetes.io/name=cert-manager-istio-csr" -n <istio_csr_project_name>
+----
+
+.. Delete the resources in {SMProductName} or Istio deployed namespace listed by running the following command:
++
+[source,terminal]
+----
+$ oc get roles.rbac.authorization.k8s.io,rolebindings.rbac.authorization.k8s.io -l "app=cert-manager-istio-csr,app.kubernetes.io/name=cert-manager-istio-csr" -n <istio_csr_project_name>
```

Comment:
@swghosh @bharath-b-rh @lunarwhite Is this technically correct? Do these commands delete the resources as described in the steps?

Reply:
@snarayan-redhat The commands are for listing the resources, and user has to delete the listed resources.

Reply:
That was my understanding as well. But the wordings doesn't seem to imply it well. 

@subhtk `Delete..by running the following command`. This command is for listing and not deleting. Deletion would be the following action. It must be something like `Fetch the list of resources...` e.g.
```
.. Delete the cluster scoped resources listed by running the following command:
+
[source,terminal]
----
$ oc get clusterrolebindings.rbac.authorization.k8s.io,clusterroles.rbac.authorization.k8s.io -l "app=cert-manager-istio-csr,app.kubernetes.io/name=cert-manager-istio-csr"
----
```
@skopacz1 Maybe you can help @subhtk fix the wordings?